sqlanalytics/resource_sql_global_config.go

```diff
@@ -0,0 +1,123 @@
+package sqlanalytics
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databrickslabs/terraform-provider-databricks/common"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type ConfPair struct {
+	Key   string `json:"key"`
+	Value string `json:"value"`
+}
+
+// GlobalConfig ...
+type GlobalConfig struct {
+	SecurityPolicy          string            `json:"security_policy,omitempty" tf:"default:DATA_ACCESS_CONTROL"`
+	DataAccessConfig        map[string]string `json:"data_access_config,omitempty"`
+	InstanceProfileARN      string            `json:"instance_profile_arn,omitempty"`
+	EnableServerlessCompute bool              `json:"enable_serverless_compute,omitempty" tf:"default:false"`
+}
+
+// GlobalConfigForRead ...
+type GlobalConfigForRead struct {
+	SecurityPolicy          string     `json:"security_policy"`
+	DataAccessConfig        []ConfPair `json:"data_access_config"`
+	InstanceProfileARN      string     `json:"instance_profile_arn,omitempty"`
+	EnableServerlessCompute bool       `json:"enable_serverless_compute,omitempty"`
+}
+
+// NewSqlGlobalConfigAPI ...
+func NewSqlGlobalConfigAPI(ctx context.Context, m interface{}) sGlobalConfigAPI {
+	return sGlobalConfigAPI{m.(*common.DatabricksClient), ctx}
+}
+
+// sAPI ...
+type sGlobalConfigAPI struct {
```

Comment:
what is `s`? :) maybe `globalConfigAPI`?

Reply:
artifact of find/replace :-)